src/services/document-service.js

```diff
@@ -104,6 +104,19 @@ class DocumentService {
     return this.documentStore.getDocumentTagsMatchingText(searchString);
   }
 
+  async findDocumentsMetadata(query) {
+    const sanitizedQuery = query.trim();
+
+    const queryConditions = [
+      { archived: false },
+      { title: { $regex: `^${sanitizedQuery}`, $options: 'i' } }
```

Comment:
When I query "Film (Kopie" I get `name: 'MongoServerError', status: 500, message: 'Regular expression is invalid: missing )', code: 51091...`, because the opening parenthesis is interpreted as RegExp special character. We need to regex-encode the input here.

Reply:
Indeed, good catch, thank you!
Fixed.